tests/test_service_account.py

```diff
@@ -76,9 +76,6 @@ def test_service_account_email(self):
     self.assertEqual(self.service_account_email,
                      self.credentials.service_account_email)
 
-  def test_create_scoped_required_without_scopes(self):
-    self.assertTrue(self.credentials.create_scoped_required())
```

Comment:
Because `create_scoped_required` arguably became somewhat pointless, and was deleted, and when added back in I forgot to put the test back in (i.e. this in particular is an oversight). I'm still unclear why we'd want to keep that method if it'll always be false, however, given that I've been told we want an automatic and transparent upgrade of service account credentials in the absence of scopes.